src/core/pipelines/segment/create_segment_loader.ts

```diff
@@ -173,35 +187,38 @@ export default function createSegmentLoader<T>(
                     IPipelineLoaderRequest |
                     IPipelineLoaderWarning >
     {
-      const request$ = backoff<ISegmentLoaderEvent<T>>(
-        tryCatch<ISegmentLoaderArguments,
-                 ISegmentLoaderEvent<T>>(loader, loaderArgument),
-        backoffOptions
-      ).pipe(
+      const request$ = (url : string | null) => {
+        const loaderArgument = objectAssign({ url }, wantedContent);
+        return observableConcat(
+          observableOf({ type: "request" as const, value: loaderArgument }),
+          tryCatch(loader, loaderArgument));
+      };
+      return tryURLsWithBackoff(wantedContent.segment.mediaURLs ?? [null],
```

Comment:
Why [null] ?

Reply:
Equivalent to saying "try a single request where the URL is not known".